### ISSUE TITLE
CRD Upgrader should avoid touching kube-prometheus-stack's CRD unless requested

### ISSUE THREAD

**Issue by kien-truong** [k8ssandra/k8ssandra#1378]
**Is your feature request related to a problem? Please describe.**
We have a separate kube-prometheus-stack deployment in our cluster, and therefore does not use the one included in K8ssandra. However, when we upgrades our K8ssandra helm chart, CRD-upgrader upgrades all kube-prometheus-stack's CRD as well, leading to conflicts with our existing kube-prometheus-stack installation.

**Describe the solution you'd like**
CRD Upgrader should avoid touching kube-prometheus-stack's CRD unless requested

**Comment by bradfordcp** [comment]
In `k8ssandra-operator` we're assuming you have a centrally managed prometheus stack and will not use ours. I'm hesitant to spend  cycles here as it shouldn't be a problem in that type of install. @burmanm what do you think?

**Comment by burmanm** [comment]
Yes, the prometheus-stack in the k8ssandra Helm default installation has always been problematic. I'd personally be happy to remove it from our autoinstall instead of fixing the CRD upgrader with options.

The CRD upgrader is not really a tricky thing to modify (upgrade.go's findCRDDir function could just skip prometheus directory and install rest) so if someone from the community wants to contribute I can happily review, but I'd still go with the nuke option and update the documentation to say "install it if you wish to enable automated metrics / Grafana dashboards".
